### ISSUE TITLE
Meeting Countdown crashes App - Pre-Release 2.0.0.5

### ISSUE THREAD

**Issue by yjenb** [AntonyCorbett/OnlyT#433]
**Describe the bug**
As soon as the meeting countdown timer starts, the app crashes

**To Reproduce**
1. Go to Options
2. Activate Meeting Countdown
2. Set a time within the duration of the countdown
3. go back to Main Screen
4. Crash

**Expected behavior**
Meeting Countdown timer would be displayed.

**From the Log**
```
2021-07-23 13:52:52.945 +02:00 [INF] Launching countdown timer
2021-07-23 13:52:52.967 +02:00 [FTL] Unhandled exception
System.NotSupportedException: Visual elements not instantiated!
   at OnlyT.CountdownTimer.CountdownControl.CheckVisualElements() in D:\ProjectsPersonal\OnlyT\OnlyT.CountdownTimer\CountdownControl.cs:line 227
   at OnlyT.CountdownTimer.CountdownControl.RegisterNames(Canvas canvas) in D:\ProjectsPersonal\OnlyT\OnlyT.CountdownTimer\CountdownControl.cs:line 235
   at OnlyT.CountdownTimer.CountdownControl.OnCanvasLoaded(Object sender, RoutedEventArgs e) in D:\ProjectsPersonal\OnlyT\OnlyT.CountdownTimer\CountdownControl.cs:line 250
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseEvent(RoutedEventArgs e)
   at System.Windows.BroadcastEventHelper.BroadcastEvent(DependencyObject root, RoutedEvent routedEvent)
   at System.Windows.BroadcastEventHelper.BroadcastLoadedEvent(Object root)
   at System.Windows.Media.MediaContext.FireLoadedPendingCallbacks()
   at System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
   at System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
   at System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
   at System.Windows.Media.MediaContext.Resize(ICompositionTarget resizedCompositionTarget)
   at System.Windows.Interop.HwndTarget.OnResize()
   at System.Windows.Interop.HwndTarget.HandleMessage(WindowMessage msg, IntPtr wparam, IntPtr lparam)
   at System.Windows.Interop.HwndSource.HwndTargetFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2021-07-23 13:52:53.096 +02:00 [INF] Settings changed and saved
2021-07-23 13:52:53.237 +02:00 [INF] ==== Exit ====
```

**Desktop (please complete the following information):**
 - Windows 10 Pro, 21H1 
 - All current updates installed

**Comment by AntonyCorbett** [comment]
@yjenb Thanks for testing! All fixed in 2.0.0.7

**Comment by yjenb** [comment]
Thanks for that quick fix. Works perfect now :-)
